### PR TITLE
#164271750 API swagger documentation 

### PR DESCRIPTION
**Pull request by Elisha-Misoi**
#### What does this PR do?
- Adds API documentation using Swagger REST Framework
#### Description of Task to be completed?
- Document API using swagger
- Add URL for swagger docs

#### How should this be manually tested?
- Clone this repository ```git clone https://github.com/jkamz/ireporter.git ``` 
- Checkout to the branch ``` git checkout ch-api-swagger-documentation-164271750```
- Cd into the api directory
- Install all requirements with ```pip install -r requirements.txt```
- Run server using the command ```python manage.py runserver```
- Go to the URL  ```<your-localhost>/api/ireporter/docs```

#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#164271750](https://www.pivotaltracker.com/story/show/164271750)

#### Screenshot
screen shot 2019-03-05 at 09 48 00

#### Questions:
N/A

### CODE REVIEW

**Comment by Elisha-Misoi** [comment]
screen shot 2019-03-05 at 09 48 00

**Comment by Siffersari** [comment]
@Elisha-Misoi You could name the branches following the format 

*{story type}-{2-3 word summary}-{pivotal tracker id}*

The {pivotal tracker id} should not include the '#'  i.e ch-API-swagger-documentation-164271750

You might have forgotten to 'Start' the story in the first commit

**Comment by Elisha-Misoi** [comment]
> @Elisha-Misoi The documentation is well done
> You could name the branches following the format
> 
> _{story type}-{2-3 word summary}-{pivotal tracker id}_
> 
> The {pivotal tracker id} should not include the '#' i.e ch-API-swagger-documentation-164271750
> 
> You might have forgotten to 'Start' the story in the first commit

Thanks @AlvinMugambi for the note. I have made the change in the branch name. I'll make the other two changes in my next PR since it doesn't affect the code base.

**Comment by Elisha-Misoi** [comment]
> The documentation is good. It describes the endpoints well. Remember not to merge the changes to master instead merge with the develop branch.

Thanks @AlvinMugambi I have made the change.

**Comment by Benkimeric** [comment]
Nice work.

**Comment by Elisha-Misoi** [comment]
> Nice work.

👍🏽

File: api/config/urls.py [hunk 1]
```diff
@@ -36,6 +40,7 @@
 
 urlpatterns = [
     path('api/', doc_urls),
+    path('api/ireporter/swagger', swagger_view),
```
Comment (meshack-mbuvi) [inline api/config/urls.py]: change the path to `path('/api/ireporter/docs',swagger_view)`

**Pull request closed without merging**